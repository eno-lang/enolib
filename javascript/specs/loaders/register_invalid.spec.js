const enolib = require('../..');

describe('register (with invalid arguments)', () => {
  describe("trying to register 'string'", () => {
    it('throws an error', () => {
      expect(() => enolib.register({ string: value => value })).toThrowErrorMatchingSnapshot();
    });
  });
});
