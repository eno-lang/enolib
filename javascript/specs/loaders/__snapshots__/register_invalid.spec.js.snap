// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`register (with invalid arguments) trying to register 'string' throws an error 1`] = `"You cannot register 'string' as a type/loader with enolib as this conflicts with the native string type accessors."`;
